src/components/app/data/hooks/useDefaultSearchFilters.test.jsx

```diff
@@ -31,10 +36,18 @@ jest.mock('react-router-dom', () => ({
 }));
 
 const mockEnterpriseCustomer = enterpriseCustomerFactory();
+const mockAuthenticatedUser = authenticatedUserFactory();
+const mockSearchCatalogs = ['test-catalog-uuid-1', 'test-catalog-uuid-2'];
 
 const SearchWrapper = (value) => function BaseSearchWrapper({ children }) {
   // eslint-disable-next-line react/jsx-filename-extension
```

Comment:
nit: given the file was renamed to the `jsx` extension, I believe this line could be removed?